docs/query_engine/roadmap.md

```diff
@@ -1,19 +1,23 @@
 # Roadmap
 
-## Launch M3 Coordinator as a bridge for the Read/Write path of M3DB into open source (Q1 2018) 
+## Q3-Q4 2019: M3QL support
+* Complete support for the M3 Query Language (M3QL)
 
-### V1 (late Q4 2017 - early Q1 2018)
-* Create a gRPC/Protobuf service.
-* Handlers for Prometheus remote read/write endpoints.
-* Perform fanout to M3DB nodes.
-* Tooling to set up M3 Coordinator alongside Prometheus.
+## Q1-Q2 2019: Graphite and Prometheus feature parity
+* Full support for Prometheus functions
+* Ingestion and querying support for Graphite
+* Tracing support using [Jaeger](https://www.jaegertracing.io/)
+* Basic cost accounting
+* Performance improvements
```

Comment:
Maybe add `Containerization improvements`? There's definitely going to be some work required here